hpi-schul-cloud/nuxt-client
# BC-5426 - Add a delete option to the school specific Privacy Policy and Terms of Use

# Short Description

QA note:
Change tested on env: 
https://nbc-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
https://brb-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
https://default-bc-5426-delete-policy-terms.cd.dbildungscloud.dev/
No issues found. All accept criteria done. 
See JIRA ticket https://ticketsystem.dbildungscloud.de/browse/BC-5426





## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-5426


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
